### ISSUE TITLE
Improve keyboard navigation and general a11y of UI shell

### ISSUE THREAD

**Issue by emyarod** [carbon-design-system/carbon#1497]
### Summary

Currently the UI shell is still lacking in accessible keyboard navigation patterns for some panels and menus.

### Justification

Improved usability for users

### Desired UX and success metrics

Users should be able to navigate the entire page without a mouse
